substrate/frame/contracts/src/lib.rs

```diff
@@ -223,7 +224,7 @@ pub struct Environment<T: Config> {
 pub struct ApiVersion(u16);
 impl Default for ApiVersion {
 	fn default() -> Self {
-		Self(4)
+		Self(5)
```

Comment:
Only bumped when the stable API gets extended.

```suggestion
		Self(4)
```